### ISSUE TITLE
Distorted LibreOffice taskbar icon

### ISSUE THREAD

**Issue by aramazano** [lxqt/lxqt#1483]
Taskbar LO icon gets subtly distorted when,

* LO is put into background *and* mouse pointer is not hovered on the LO icon at taskbar. When the pointer is moved over to LO icon (without clicking, LO still in the BG) the the distortion goes away.

* In case multiple LO windows are open, then hovering the mouse pointer on the LO task pops up the group (of windows) list. In this group list, some or all of the icons are shown distorted. It depends on mouse hovering. The icon in the pop-up list is always good if mouse pointer is hovering on it. But it is *sometimes* distorted if mouse pointer is not on it.

Here are the screenshots:

LO in background and mouse pointer is NOT on LO icon:
![lo-bg1](https://user-images.githubusercontent.com/34963702/39496457-30e58dbe-4da7-11e8-884c-4127f828f759.png)

LO in background and mouse pointer IS on LO icon:
![lo-bg2](https://user-images.githubusercontent.com/34963702/39496473-4521d314-4da7-11e8-97e7-179b06e2e540.png)

LO in foreground and pointer is NOT on LO icon:
![lo-fg1](https://user-images.githubusercontent.com/34963702/39496487-51590f80-4da7-11e8-95b2-cfc192062af4.png)

LO in foreground and pointer IS on LO icon:
![lo-fg2](https://user-images.githubusercontent.com/34963702/39496497-5f32ad78-4da7-11e8-9a1b-653faccfbf81.png)

Sorry I forgot to include the pointer in the screenshots, but you can infer where the pointer is from the border drawn around the taskbar buttons.

##### System Information
* Distribution & Version: LXQt [experimental-snapshots](https://lxqt.debian.net/) on top of Debian Testing (Buster)
* Kernel: 4.9.0-4-amd64
* Qt Version: 5.10.1
* liblxqt Version: liblxqt0 / 0.12.1~8-g80f9324-1
* lxqt-panel Version: 0.12.1~30-g577e7113-1

**Comment by tsujan** [comment]
What happens when you enable compositing? Contrary to common beliefs, compositing is good for performance. However, I just want to check a hypothesis with that question.

**Comment by tsujan** [comment]
Never mind! I think I found the reason. You use gnome icon theme or some icon set that uses gnome icons for libreoffice. See the real icon here:

![icon](https://user-images.githubusercontent.com/981076/39497011-66867604-4db6-11e8-88f9-404f7fe0ba39.png)

Use another icon set with good LibreOffice icons. Closing this.

**Comment by tsujan** [comment]
Another example from gnome-icon-theme:

![icon1](https://user-images.githubusercontent.com/981076/39497088-d977f6d8-4db6-11e8-977d-71aa3cc7880f.png)

**Comment by tsujan** [comment]
I searched and found those corrupt icons belong to LibreOffice itself and are also installed inside hicolor, which Oxygen uses because it lacks LibreOffice icons. So, it isn't related to gnome-icon-theme.

You could use the Breeze icon theme instead, which is more elegant than oxygen. There are also so many good icon sets on the Internet.

**Comment by aramazano** [comment]
I am using Oxygen icon theme. Tried every other icon theme and only "high contrast" theme is showing LO icons always correct. Other themes are worse than oxygen in that they even distort main menu LO icons. In all the cases where LO icons get distorted, the same "Gnome" icons are being used. E.g. when I change icon theme, normally all the icons are changed everywhere according to the theme - but not for LO. The LO icon set is always the same (Gnome style as you say), except in high contrast theme, which has its own LO icons, and which has no problem with LO icons.

In summary:

High Color: Uses its own LO icon set for LO, always presents them correct both in main manu and in taskbar.

Oxygen and Adwaita: Uses gnome icon set for LO, presents them correctly in main menu, distorts sometimes in taskbar.

Tango and Gnome: Uses gnome icon set for LO, presents them always distorted in main menu, distorts sometimes in taskbar.

I am fine with occasionally distorted LO icons. No big deal. If it is something outside LXQt domain then let us forget about it.

**Comment by tsujan** [comment]
> I am using Oxygen icon theme

Please read my last comment above; I had ESP ;)

**Comment by tsujan** [comment]
And a short rational explanation:

For some reason that I can only guess (our icon engine), the existing SVG icons are prioritized. But they are corrupt in the case icons that come with LibreOffice.

Using an SVG icon belonging to a theme is not only valid but also good because SVG icons don't get blurry.
